src/Components/ImageOrVideo.tsx

```diff
@@ -26,10 +26,10 @@ export const ImageOrVideo = connect(function ImageOrVideo<T extends string>({
   }>
 }) {
   const [isPaused, setIsPaused] = useState(false)
-  const [shouldAutoplay, setShouldAutoplay] = useState(true)
+  const [shouldAutoplay, setShouldAutoplay] = useState(false)
```

Comment:
* [ ] FYI Looking at the reducedMotion logic, the initial state of `isPaused` looks a bit contradictory now.

Reply:
It was set on false on purpose: With this I wanted to avoid a slight "button available" flicker, when the page is initially rendered.